src/projections.jl

```diff
@@ -211,7 +212,7 @@ function projection_gradient_on_set(::DefaultDistance, v::AbstractVector{T}, ::M
     λ, U = LinearAlgebra.eigen(X)
 
     # if all the eigenvalues are >= 0
-    if all(>=(0), λ)
+    if all(λ .≥ zero(T))
```

Comment:
why this change? This seems to be allocating when the first version was not

Reply:
strangely this breaks on my desktop now (worked earlier) and in this test too - https://travis-ci.com/github/matbesancon/MathOptSetDistances.jl/jobs/427137353